From 1942ffc4320f8204a634259bc8b0cf9c645a2be3 Mon Sep 17 00:00:00 2001
From: Terra <TerraTech@users.noreply.github.com>
Date: Tue, 12 Jun 2018 04:46:49 -0400
Subject: [PATCH] php.syntax: Add syntax highlighting for keyword: 'null'

  Even though there are other 'NULL' keywords highlighted with 'white',
    I followed the 'true'/'false' keyword color style.
---
 misc/syntax/php.syntax | 1 +
 1 file changed, 1 insertion(+)

diff --git a/misc/syntax/php.syntax b/misc/syntax/php.syntax
index 453c0c07c..5a8506a27 100644
--- a/misc/syntax/php.syntax
+++ b/misc/syntax/php.syntax
@@ -35,6 +35,7 @@ context default
     keyword whole include brightmagenta
     keyword whole include_once brightmagenta
     keyword whole new brightmagenta
+    keyword whole null brightmagenta
     keyword whole private brightmagenta
     keyword whole protected brightmagenta
     keyword whole public brightmagenta
